tldw_Server_API/app/core/Web_Scraping/Article_Extractor_Lib.py

```
@@ -1645,7 +1646,7 @@ def _run_with_retries(
                 return None, exc, attempts
             delay_s = (base_delay_ms / 1000.0) * (2 ** attempts)
             if jitter_ms:
-                delay_s += random.uniform(0.0, jitter_ms / 1000.0)
+                delay_s += random.uniform(0.0, jitter_ms / 1000.0)  # nosec B311
```

Ruff (0.15.6)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)